lib/decoder.js

```diff
@@ -84,7 +84,9 @@ class Decoder {
     this.signature = signature;
     this.algorithm = this.getAlgorithm();
 
-    this.verifySignature(encodedHeader, encodedBody);
+    if(_key != null) {
```

Comment:
Could you please use a strict comparison here (`!==`)?

Reply:
Fixed in 5ac188ab8b86009bbdba081a755eb74f9e3a194c